packages/electron-commons/src/initializers/electron-node.ts

```diff
@@ -12,14 +12,39 @@ export interface InitializeNodeEnvironmentOptions extends InitializerOptions {
     processOptions?: Pick<SpawnOptions, 'cwd' | 'shell' | 'env' | 'stdio' | 'windowsHide'>;
 }
 
+/**
+ * Detailed info about process exit.
+ */
+export interface ProcessExitDetails {
+    /**
+     * The exit code if the child exited on its own
+     */
+    exitCode: number | null;
+
+    /**
+     * The signal by which the child process was terminated
```

Comment:
If the process traps the signals and exits, is it terminated by the signal?

Like with the previous property, not having a description would give me more clarity.

Reply:
this is taken from node documentation about child exit event